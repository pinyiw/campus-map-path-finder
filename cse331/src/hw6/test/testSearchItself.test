# Load graph
LoadGraph staffs staffSuperheroes.tsv

# Find path
FindPath staffs Ernst-the-Bicycling-Wizard Ernst-the-Bicycling-Wizard