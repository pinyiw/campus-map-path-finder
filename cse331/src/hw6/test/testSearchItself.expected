# Load graph
loaded graph staffs

# Find path
path from Ernst-the-Bicycling-Wizard to Ernst-the-Bicycling-Wizard: